components/gallery/List.vue

```diff
@@ -0,0 +1,303 @@
+<template>
+    <div class="gallery-list">
+        <ul class="list list-main">
+            <li
+                v-for="(main) in preparedList"
+                class="item-main"
+            >
+                <ul
+                    v-for="(list) in main"
+                    class="list"
+                >
+                    <li
+                        v-for="item in list.listItems"
+                        :key="item.id"
+                        class="item"
+                    >
+                        <h3
+                            class="item-title"
+                            @mouseout="mouseOut()"
+                            @mouseover="mouseOver(item)"
+                        >
+                            <nuxt-link
+                                :to="item.uri"
+                                class="link"
+                            >
+                                {{ item.title }}
+                            </nuxt-link>
+                        </h3>
+                    </li>
+                </ul>
+            </li>
+        </ul>
+
+        <transition
+            mode="out-in"
+            name="fade"
+        >
+            <wp-image
+                v-if="image"
+                :key="image.id"
+                :alt="image.altText"
+                :caption="caption"
```

Comment:
These are done by `wp-image` already

Reply:
I removed the `alt` tag but I wanted to use the caption incase both the video and image had errors then I could display the `caption` as a fallback.
![image](https://user-images.githubusercontent.com/1283934/141696301-33856796-62f1-4520-80d6-0c4a6bea9ddc.png)